integration_tests/cleanup-jest-html-reports.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Set the number of directories to keep
+ num_of_dir_to_keep=$1
+
+ if [[ $GITHUB_WORKFLOW ]]; then
+    cd $GITHUB_WORKSPACE
+    cd jest-html-report
+    ls
+ else
+   cd ./reports/jest-html-report
+   ls
+fi
+
+# List all directories and sort by modification time in descending order
+dirs=( $(ls -d -t */ | tail -n +$((num_of_dir_to_keep+1))) )
```

Comment:
Thanks updated the matching pattern to avoid unintended deletions. Outer parentheses required if I remove the deletion is not successful listing all dirnames and giving no such file or directory

Reply:
sorry without parentheses its working removed it now